### DISCUSSION TITLE
2.2.8版本的Carousel有点问题啊

### DISCUSSION THREAD

**Discussion by hsk1995** [vueComponent/ant-design-vue#4689] [Ideas]
![image](https://user-images.githubusercontent.com/40809868/134634800-f06b749b-ce56-4bd7-a9fe-a860453cb1bd.png)
左侧的箭头没有插进去
另外希望可以增加一个自定义样式的属性   现在这样改动样式有点不方便

**Comment by zkwolf** [comment]
看起来是demo的style写的有问题
